scripts/run-clang-format.sh

```diff
@@ -27,20 +27,35 @@ shift
 APPLY_FIXES=$1
 shift
 
-# clang format will only find its configuration if we are in 
+# clang format will only find its configuration if we are in
 # the source tree or in a path relative to the source tree
 pushd $SOURCE_DIR
 
+src=$SOURCE_DIR
+SOURCE_PATHS=($src/tiledb $src/test $src/examples $src/tools $src/experimental)
+FIND_FILES=(-name "*.cc" -or -name "*.c" -or -name "*.h")
+
 if [ "$APPLY_FIXES" == "1" ]; then
   $CLANG_FORMAT -i $@
 
 else
+  NUM_CORRECTIONS=`find "${SOURCE_PATHS[@]}" "${FIND_FILES[@]}" -print0 | xargs -0 -P8 $CLANG_FORMAT -output-replacements-xml | grep offset | wc -l`
```

Comment:
pre-existing but ... noticed that clang-format-9 --help shows a '--output-replacement-xml' option (two leading dashes), but not one with a single leading dash - though it nevertheless seems to accept the single leading dash.
(No clue if anything would work differently (worse) if the second leading dash was added.)